corehq/apps/users/views/mobile/users.py

```diff
@@ -650,6 +650,7 @@ def _format_user(self, user_json):
             'last_name': user.last_name,
             'phoneNumbers': user.phone_numbers,
             'user_id': user.user_id,
+            'location': user.sql_location.to_json() if include_location and user.sql_location else None,
```

Comment:
i went back and forth on whether or not to batch this location get by all the users returned. however since it's only ever 10 users and only used in the login as feature, i opted for doing the lookup on each individual user for readability